node-api/backend/utils/validator.go

```diff
@@ -41,3 +45,51 @@ func ValidatorIndexByID(st *statedb.StateDB, keyOrIndex string) (math.U64, error
 	}
 	return st.ValidatorIndexByPubkey(key)
 }
+
+// GetValidatorStatus returns the current validator status based on its set
+// Epoch values.
+func GetValidatorStatus(epoch math.Epoch, validator *types.Validator) (string, error) {
```

Comment:
i understand this is from Prysm, but I feel either we should have a sanity check UT for this function here OR put these functions on the Validator type and UTs there

Reply:
Added `Status` function on Validator type and added unit tests.